lib/nimble.phx.gen.template/addons/ex_coveralls.ex

```diff
@@ -13,9 +13,11 @@ defmodule Nimble.Phx.Gen.Template.Addons.ExCoveralls do
   end
 
   defp copy_files(%Project{} = project) do
+    # There is no test for `lib/*/router.ex` on the new Phoenix app, so that reduce the coverage to 77.8%
+    # We are setting the minimum_coverage as 77 to work with the new Phoenix app
```

Comment:
![image](https://user-images.githubusercontent.com/11751745/94550988-d2c20e80-027e-11eb-832d-86adb00c5dfc.png)

Reply:
It looks kind of fragile, to be honest. What kind of tests would this require to keep it at 100%?

Reply:
We need to add the test for `api pipeline` and the LiveDashboard @olivierobert 

![Screen_Shot_2020-10-01_at_11_08_08](https://user-images.githubusercontent.com/11751745/94767414-7bcf4d00-03d6-11eb-94c3-c0f5e230f25b.png)

Reply:
Fixed that in https://github.com/nimblehq/elixir-templates/pull/24/commits/519207309e21d483140a765a3412205d2ec48999 @olivierobert